docs/reference/query-dsl/query-string-query.asciidoc

```diff
@@ -83,6 +83,9 @@ to `AUTO`. See <<fuzziness>> for allowed settings.
 |`fuzzy_prefix_length` |Set the prefix length for fuzzy queries. Default
 is `0`.
 
+|`fuzzy_transpositions` |Set to `true` to enable fuzzy transpositions (`ab` -> `ba`).
```

Comment:
I'd say "set `false` to disable fuzzy transpositions" because the default is true ?